setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -968,21 +968,29 @@ def recurLayerCollection(layerColl):
         bpy.ops.object.mode_set(mode='POSE')
 
     # Function to automatically move a bone (if it exists) to the specified bone layer. pass in num+1 than layer desired.        
-    def move_bone(bone_name,to_layer):
+    def move_bone(bone_name,to_layers):
         armature =  bpy.context.active_object
         armature_data = armature.data
         
         if bone_name in armature_data.bones:
             bone = armature_data.bones[bone_name]
             
-            # Set to true only where in bone layer we care about
-            bone.layers = [i == to_layer-1 for i in range(32)]  # Whatever BL is, -1.
+            for i in range(32):
+                bone.layers[i] = False
+            
+            to_layers = [to_layers] if isinstance(to_layers, int) else to_layers
+            
+            for layer in to_layers:
+                bone.layers[layer] = True
+                print("enabling layer: " + str(layer) + " for " + bone_name)
+            
+
 
     if not is_version_4:
         # Put away every other bone to the physics layer (22)
         for bone in bpy.context.active_object.data.bones:
             if bone.layers[0]:  # Check if the bone is on layer face
-                move_bone(bone.name, 26)  # Move the bone to layer 23
+                move_bone(bone.name, 25)  # Move the bone to layer 23

```

Comment:
Same thing here, I think this comment is stale now, might want to take a note to update it in the future.